src/styles/index.scss

```diff
@@ -1,3 +1,3 @@
-body {
-  margin: 0;
-}
+@import './utils/variables';
```

Comment:
Make sure that the file `./utils/variables.scss` exists and contains the necessary variable definitions, such as `$main-color` and `$secondary-color`, which are used in your styles.